test/LibraryManager.Test/CacheServiceTest.cs

```diff
@@ -0,0 +1,238 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Web.LibraryManager.Contracts;
+using Microsoft.Web.LibraryManager.Mocks;
+using Microsoft.Web.LibraryManager.Providers.Cdnjs;
+using Microsoft.Web.LibraryManager.Providers.FileSystem;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Web.LibraryManager.Test
+{
+    [TestClass]
+    public class CacheServiceTest
+    {
+
+        private string _filePath;
+        private string _cacheFolder;
+        private string _projectFolder;
+        private string _catalogCacheFile;
+        private IDependencies _dependencies;
+        private HostInteraction _hostInteraction;
+        private CacheService _cacheService;
+
+        [TestInitialize]
+        public void Setup()
+        {
+            _cacheFolder = Environment.ExpandEnvironmentVariables(@"%localappdata%\Microsoft\Library\");
```

Comment:
This is Windows specific?  Do we need to do something different for other OSs?

Reply:
Also, are the unit tests working against the user's live cache?  That seeems... dangerous... if there's a bug?

Reply:
1. Probably, but we haven't ran these tests on other OS's as far as I know.
2. No, this is not the users live cache.